src/main/java/gmbs/domain/state/finish/BlackJack.java

```diff
@@ -0,0 +1,24 @@
+package gmbs.domain.state.finish;
+
+import gmbs.domain.card.CardHand;
+import gmbs.domain.game.result.Result;
+import gmbs.domain.state.CardState;
+
+public final class BlackJack extends Finish {
+
+    private BlackJack(final CardHand cardHand) {
+        super(cardHand);
+    }
+
+    public static BlackJack from(final CardHand cardHand) {
+        return new BlackJack(cardHand);
+    }
+
+    @Override
+    public Result getCompareResult(CardState compareCardState) {
+        if (compareCardState.isSameCardStateWith(BlackJack.class)) {
+            return Result.DRAW;
+        }
+        return Result.WIN;
```

Comment:
```suggestion
        if (compareCardState.isBlackJack()) {
            return Result.DRAW;
        }
        return Result.WIN;
```
위와 같은 방식이 더 좋을 것 같네요~ 굳이 타입에 대한 정보를 넣어주지 않고 CardState에게 물어보는 것이 더 객체지향적이라고 생각해요~